docs/integrations/builtin/app-nodes/n8n-nodes-base.googlecalendar.md

```diff
@@ -17,18 +17,24 @@ Refer to [Google Calendar credentials](/integrations/builtin/credentials/google/
 
 ## Operations
 
-* Calendar
-    * If a time-slot is available in a calendar
-* Event
-    * Add a event to calendar
-    * Delete an event
-    * Retrieve an event
-    * Retrieve all events from a calendar
-    * Update an event
+* **Calendar**
+    * **Availability**: If a time-slot is available in a calendar
+* **Event**
+    * **Create**: Add a event to calendar
+    * **Delete**: Delete an event
+    * **Get**: Retrieve an event
+    * **Get Many**: Retrieve all events from a calendar
+    * **Update**: Update an event
 
 ## Templates and examples
 
 <!-- see https://www.notion.so/n8n/Pull-in-templates-for-the-integrations-pages-37c716837b804d30a33b47475f6e3780 -->
 [[ templatesWidget(title, 'google-calendar') ]]
 
---8<-- "_snippets/integrations/builtin/app-nodes/operation-not-supported.md"
+## Related resources
+
+n8n provides a trigger node for Google Calendar. You can find the trigger node docs [here](/integrations/builtin/trigger-nodes/n8n-nodes-base.n8n-nodes-base.googlecalendartrigger.md/).
+
+Refer to [Google Calendar's documentation](https://developers.google.com/calendar/api/v3/reference){:target=_blank .external-link} for more information about the service.
+
+View [example workflows and related content](https://n8n.io/integrations/_Name_/){:target=_blank .external-link} on n8n's website.
```

Comment:
Now that we have the "Templates and examples" section, we shouldn't need this link in the Related resources section. (I think Debs had already removed it from the most recent template). Can you remove here?